app/javascript/app/components/ndcs/ndcs-explore-table/ndcs-explore-table-selectors.js

```diff
@@ -78,6 +79,29 @@ export const tableGetFilteredData = createSelector(
   }
 );
 
+export const getSelectedIndicatorHeader = createSelector(
+  [getMapIndicator],
+  selectedIndicator => {
+    if (!selectedIndicator) return null;
+    return `${selectedIndicator.label} (Current selection)`;
+  }
+);
+
+const addIndicatorColumn = createSelector(
+  [tableGetFilteredData, getMapIndicator, getSelectedIndicatorHeader],
+  (data, selectedIndicator, selectedIndicatorHeader) => {
+    if (!data || isEmpty(data)) return null;
+    const updatedTableData = data;
+    return updatedTableData.map(countryRow => {
+      const updatedCountryRow = { ...countryRow };
+      const countryIndicatorData = selectedIndicator.locations[countryRow.iso];
+      updatedCountryRow[selectedIndicatorHeader] =
+        countryIndicatorData && countryIndicatorData.value;
+      return updatedCountryRow;
+    });
+  }
+);
+
```

Comment:
Could you move it to a shared selectors file? Because we're keeping duplicating the code for ndcs and lts table :busts_in_silhouette: